bevy_rapier3d/Cargo.toml

```diff
@@ -53,14 +53,32 @@ log = "0.4"
 serde = { version = "1", features = ["derive"], optional = true }
 
 [dev-dependencies]
+benches_common = { path = "../benches_common" }
 bevy = { version = "0.14.0-rc.3", default-features = false, features = [
     "x11",
     "tonemapping_luts",
     "bevy_state",
 ] }
 approx = "0.5.1"
 glam = { version = "0.27", features = ["approx"] }
+divan = "0.1"
+
+[dev-dependencies.rapier3d]
+features = ["profiler"]
+version = "0.20"
```

Comment:
I don't really like that, as it adds it for unrelated use-cases like examples, but I'm not sure if I can specify that only for benches.

I imagine the best solution is to create a separate project for benches.